tests/std/tests/P1222R4_flat_set/test.cpp

```diff
@@ -166,6 +273,76 @@ void test_constructors() {
     assert_all_requirements_and_equals(flat_multiset<int>(std::move(b), allocator<int>{}), {-1, 1, 2, 7, 7, 7, 100});
 }
 
+template <iterator_pair_construction Choice>
+void test_allocator_extended_constructors() {
+    using vec = alternative_vector<int, allocator<int>, Choice>;
+    constexpr allocator<int> ator;
+    constexpr std::less<int> comp;
+    {
+        using fs = flat_set<int, std::less<int>, vec>;
+
+        fs s0{1, 1, 2, 3, 5, 8};

```

Comment:
I find it somewhat confusing that the expected-result container `fs s0` is being constructed from `{1, 1, 2, 3, 5, 8}` which will lead to it storing just `1, 2, 3, 5, 8`. I would find it easier to understand if it were constructed from exactly what it ends up storing.

Reply:
Now I'm comparing the results with `s_expected` which is constructed with elements listed in the expected order.